test-data/heat_equation/heat_equation.py

```diff
@@ -9,6 +9,8 @@
 import numpy.typing as npt
 from definition import dx, k_end, k_start, nx, obs_coordinates, obs_times, u_init
 
+from ert.field_utils.grdecl_io import export_grdecl, import_grdecl
```

Comment:
How much runtime does this add?

Reply:
This runs in 2min while the version on main runs in 40s. You mentioned that it might be because `import ert` is slow right?